app/models/job_invocation.rb

```diff
@@ -40,6 +40,8 @@ class JobInvocation < ActiveRecord::Base
 
   default_scope -> { order('job_invocations.id DESC') }
 
+  validates :description_format, :length => { :maximum => 255 }, :if => :present?
+
```

Comment:
I didn't know a validator like that existed, changed to validating/truncating based on the column's length